README.md

```diff
@@ -25,21 +50,40 @@ $ npm -v && node -v
 v16.19.0
 ```
 
-You also need to install the dependencies required by React Native.  
+> **Note**
+> You also need to install the dependencies required by React Native.  
 Go to the [React Native environment setup](https://reactnative.dev/docs/environment-setup), then select `React Native CLI Quickstart` tab.  
 Follow instructions for your given `development OS` and `target OS`.
 
-## Getting Started
 
-## Quick Start
+> **Note**
+> If you're using Linux Bash for Windows, [see this guide](https://www.howtogeek.com/261575/how-to-run-graphical-linux-desktop-applications-from-windows-10s-bash-shell/) or use `node` from the command prompt.
 
-- Init project
 
-  ```bash
-  npx react-native init YourProjectName --template rn-base-project-ts
-  ```
 
-  [Check out the full documentation!](#documents)
+## Quick Start
+
+1. Init project
+    ```sh
+      npx react-native init YourProjectName --template rn-base-project-ts
+    ```
+2. Install node_modules
+   ```sh
+    yarn install
+   ```
+3. Run script install pod via package.json
+   ```sh
+    yarn postinstall
```

Comment:
I don't think we need this, `yarn postinstall` is run automatically after we run `yarn install`

Reply:
oke updated